src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ModelRunTests.kt

```diff
@@ -59,6 +59,15 @@ class ModelRunTests : SecureIntegrationTests()
         assertSuccess(responseEntity, "ModelSubmitResponse")
     }
 
+
+    @Test
+    fun `can validate model options`()
+    {
+        val entity = getModelRunEntity()
+        val responseEntity = testRestTemplate.postForEntity<String>("/model/validate/options/", entity)
+        Assertions.assertThat(responseEntity.statusCode).isEqualTo(HttpStatus.BAD_REQUEST)
```

Comment:
This test doesn't actually test anything - there could be any number of reasons for returning this status, including this endpoint being wrongly wired up!

I'd like to see a test where the response is a 200 and we get back a response that conforms to the `ModelOptionsValidate` schema.